app/server/index.js

```diff
@@ -174,10 +172,13 @@ app.post('/twilio', function(req, res) {
                                 })
                                 .catch(function(error){
                                     console.log("Storage FAILURE: ", error);
-                                    res.set('Content-Type', 'text/xml');
-                                    res.send(xml({"Response":""}));
+                                    res.sendStatus(422);
```

Comment:
Duh ? Need to catch up on this technical joke :p, i smell there is something to learn here :)